### ISSUE TITLE
Bring over parsing design

### ISSUE THREAD

**Issue by thehabes** [CenterForDigitalHumanities/TPEN-Newberry-Italian#22]

**Comment by thehabes** [comment]
Newberry Design:

![nd](https://cloud.githubusercontent.com/assets/3287006/25201929/211dbd14-2519-11e7-81d6-1fa32ef11ae6.png)

T-PEN Design:

![td](https://cloud.githubusercontent.com/assets/3287006/25201941/2a5ef35c-2519-11e7-8dc1-a0dec18d51d9.png)

**Comment by cubap** [comment]
I would start with just imitating the style, without rewriting any of the
parsing code. That way, if we need to back out of trying to change the
functions (like taking delete out of merge or separating create and
adjust), we've already got a good looking update to offer.

Patrick Cuba
Walter J. Ong S.J. Center for Digital Humanities
Saint Louis University
314-977-4249

On Wed, Apr 19, 2017 at 3:59 PM, Bryan Haberberger <notifications@github.com

**Comment by thehabes** [comment]
Yes, but to implement the design fully  I have to change the parsing code because merge line and delete line are separated in TPEN but not Newberry, which cascades a little bit into other functions (like applyRuler()).  However, if we decide we don't want to get into that, then we just get rid of "remove last line" from the design and I should be able to implement the design and the only change in the parsing code will be to connect the icons to the buttons, nothing in the logic.  Perhaps we don't even want to change the design, and we just want to do the user friendly UX upgrades we did for it (#21) .  Then, we don't have to alter any parsing code at all, just some CSS and cursor classing.   It is still a bit up in the air how deep we wanted to go with this.  This Epic is for that conversation.

**Comment by Domhnall101** [comment]
Bryan and Patrick are right ,all we want to do is the UX changes the merge and delete last line can stay together as a button. There will only every be a few users using this so If we end up with time for anything at the end it wont be this. 
Once Bryan has done that any work on this section will be bug fixing only
I'll do digital comp for styling reference

**Comment by Domhnall101** [comment]
use the same size of button, text and icons as in tpen 28
![screen shot 2017-04-21 at 9 46 34 am](https://cloud.githubusercontent.com/assets/17204541/25282893/7f329eb0-2677-11e7-9189-e2ad621a8b0f.png)
